compiler/src/iree/compiler/Dialect/Flow/Transforms/test/collapse_linalg_generic_on_tensors.mlir

```diff
@@ -0,0 +1,326 @@
+// RUN: iree-opt --split-input-file --verify-diagnostics --pass-pipeline="builtin.module(func.func(iree-flow-form-dispatch-regions{aggressive-fusion=true}))" %s | FileCheck %s
+!type = tensor<2x4x8x16x32x64xf32>
+util.global private @"__transpose_10_input" {noinline} = dense<1.0> : !type
+
+func.func @collapse1() -> !type {
+  %cst = arith.constant 0.000000e+00 : f32
+  %c0 = arith.constant 0 : index
+  %input_ptr = util.global.address @"__transpose_10_input" : !util.ptr<!type>
+  %input = util.global.load.indirect %input_ptr : !util.ptr<!type> -> !type
+  %output = tensor.empty() : !type
+  
+  // Can collapse All (d0, d1, d2, d3, d4, d5)
+  %6 = linalg.generic { indexing_maps = [ 
+            affine_map<(d0, d1, d2, d3, d4, d5) -> (d0, d1, d2, d3, d4, d5)>, 
+            affine_map<(d0, d1, d2, d3, d4, d5) -> (d0, d1, d2, d3, d4, d5)>],
+            iterator_types = ["parallel", "parallel", "parallel", "parallel", "parallel", "parallel"]
+         }
+    ins(%input : !type) outs(%output : !type) {
+    ^bb0(%arg1: f32, %arg2: f32):
+      linalg.yield %arg1 : f32
+    } -> !type
+  return %6: !type
+
+}
+//      CHECK: #[[MAP:.+]] = affine_map<(d0) -> (d0)>
+//      CHECK: %[[IN:.+]] = tensor.collapse_shape %[[INPUT:.+]] {{\[}}[0, 1, 2, 3, 4, 5]] : tensor<2x4x8x16x32x64xf32> into tensor<2097152xf32>
```

Comment:
To make debug easy you want to check the function name so that you are not matching something from another function by accident.